src/Apps/Artwork/Components/ArtistInfo.tsx

```diff
@@ -10,6 +9,7 @@ import {
   SelectedExhibitions,
   SelectedExhibitionsProps,
 } from "Styleguide/Components"
+import { EntityOverline } from "Styleguide/Components/EntityOverline"
```

Comment:
Is the idea of making it very generic here is to reuse it for Partner Info above in the design? The name seem to be quite non descriptive. Maybe if this component is repeated for several pages/sections we can chat with @design team on some naming convention here.

Reply:
This is based on a spec that @briansw came up: zpl.io/VKl7LWE

Reply:
I think that the word "Overline" is a bit confusing outside of the design world but we can all learn 😄

Reply:
I'm all totally for learning this! My question was more about the definition of this overline. Should it be ArtistOverline or more generic EntityOverline? Seems like in this case we need to pass the follow button action somehow. We can not assume that it's an artist Follow there.

But also so far seems like the same design is in PartnerProvidedInfo above and ArtistInfo. But there was a comment during the review that we might actually want to distinguish Partner from Artist more here. So I was wondering if this overline style will be reused.

Reply:
I believe the idea is that this will be reused